wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -218,6 +218,56 @@ public ParallelRaceGroup onlyWhile(BooleanSupplier condition) {
     return until(() -> !condition.getAsBoolean());
   }
 
+  /**
+   * Decorates this command to run once a condition becomes true.
```

Comment:
I feel like it should be clarified that the composition still runs (in terms of requirements etc) until the condition as well; distinguish this from triggers.